DisplayVar ---> Display

mathisonian opened the pull request:
This fixes #17 

```
[DisplayVar var:varName /]

becomes

[Display value:varName /]
```

mathisonian committed:
displayvar -> display

mathisonian committed:
display

bclinkinbeard commented:
Seems like the prop name should stay var

________________________________
From: Matthew Conlen <notifications@github.com>
Sent: Saturday, July 22, 2017 7:47:09 PM
To: idyll-lang/idyll-default-components
Cc: Subscribed
Subject: [idyll-lang/idyll-default-components] DisplayVar ---> Display (#20)


This fixes #17<https://github.com/idyll-lang/idyll-default-components/issues/17>

[DisplayVar var:varName /]

becomes

[Display value:varName /]


________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/idyll-lang/idyll-default-components/pull/20

Commit Summary

  *   displayvar -> display
  *   display

File Changes

  *   R components/display.js<https://github.com/idyll-lang/idyll-default-components/pull/20/files#diff-0> (7)
  *   M components/index.js<https://github.com/idyll-lang/idyll-default-components/pull/20/files#diff-1> (2)

Patch Links:

  *   https://github.com/idyll-lang/idyll-default-components/pull/20.patch
  *   https://github.com/idyll-lang/idyll-default-components/pull/20.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/idyll-lang/idyll-default-components/pull/20>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AAIkN6HWwzvtGZ1jZ9HnxkawiTT_WDAtks5sQon8gaJpZM4OgT-z>.

mathisonian commented:
I'd argue for `value` since its not really specific to outputting a `var`, it will display any value passed (but happens to be especially useful for displaying vars). And since `Dynamic` and `Range` already use the `value` property it feels a little more consistent

```
[Dynamic value:myVar /]
[Range value:myVar /]
```

mathisonian committed:
fix tests

mathisonian committed:
better type support. also take in var prop

mathisonian commented:
I updated so that `[Display var:x /]` still works, and added some additional type checking so that it can do reasonable things when objects, strings, and numbers are passed in.
